code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2595,7 +2595,7 @@
 		var/datum/organ/external/chest/C = H.get_organ(LIMB_CHEST)
 		for(var/datum/organ/internal/I in C.internal_organs)
 			if(I.damage > 0)
-				I.damage -= 0.20
+				I.damage = max(0,I.damage-=0.20)
```

Comment:
You're doing `max(0,I.damage-=0.20)`, you probably want to do `max(0, I.damage - 0.20)`. Note the accidental `-=`

Reply:
thanks for catching that. Coding at 3AM with a christmas headache is not recommendable to anyone.